projects/components/src/datagrid/datagrid.component.spec.ts

```diff
@@ -73,276 +72,293 @@ describe('DatagridComponent', () => {
             declarations: [HostWithDatagridComponent, DatagridDetailsComponent, DatagridDetailsPaneComponent],
         }).compileComponents();
 
-        this.finder = new WidgetFinder(HostWithDatagridComponent);
-        this.vcdDatagrid = this.finder.find(GridWithBoldRenderer);
-        this.clrGridWidget = this.vcdDatagrid.clrDatagrid;
-        this.component = this.finder.find(VcdDatagridWidgetObject).component as DatagridComponent<MockRecord, RecordId>;
+        this.finder = new AngularWidgetObjectFinder(HostWithDatagridComponent);
+        // @ts-ignore
+        this.vcdDatagrid = this.finder.find(VcdDatagridWidgetObject);
+        this.clrGridWidget = this.vcdDatagrid.findClrDatagrid();
+        this.component = this.vcdDatagrid.unwrap().elements[0].componentInstance as DatagridComponent<
```

Comment:
This line is used in 11 places `unwrap().elements[0].componentInstance` and looks hackish. I think we need to figure out a way to access the component instance more easily from the tests. 

We probably have to chat offline about this